crates/shielder_bindings/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+#![cfg_attr(not(feature = "build-uniffi"), no_std)]
+
+#[cfg(feature = "build-uniffi")]
+uniffi::setup_scaffolding!();
+
+#[cfg(feature = "multithreading-wasm")]
+pub use wasm_bindgen_rayon::init_thread_pool;
+
+extern crate alloc;
```

Comment:
I think we should include building this crate with different feature flags in our CI (probably by just calling Makefile targets)